2-mandatory/3-financial-times.js

```diff
@@ -14,7 +19,13 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+  for (let shortestTitle of allArticleTitles) {
+    let fewestWordTitle = shortestTitle.split(" ");
+    if (fewestWordTitle.length <= 7) {
```

Comment:
Why is the number **7** used to determine the fewest words?
This might be a case of **magic numbers** and should be avoided when coding.
If a new title, with a even shorter title is introduced, will this function still work?